pkg/cache/context_test.go

```diff
@@ -50,15 +50,54 @@ import (
 )
 
 const (
-	Host1  = "HOST1"
-	appID1 = "app00001"
-	appID2 = "app00002"
-	appID3 = "app00003"
-
-	pod1UID      = "task00001"
-	taskUID1     = "task00001"
-	pod1Name     = "my-pod-1"
+	Host1 = "host0001"
+	Host2 = "host0002"
+
+	appID1       = "app00001"
+	appID2       = "app00002"
+	appID3       = "app00003"
+	appID4       = "app00004"
+	appID5       = "app00005"
+	appIDUnknown = "app-none-exist"
+
+	uid1       = "uid_0001"
+	uid2       = "uid_0002"
+	uid3       = "uid_0003"
+	uid4       = "uid_0004"
+	uid5       = "uid_0005"
+	uidUnknown = "uid_0000"
+
+	pod1UID     = "task00001"
+	pod2UID     = "task00002"
+	taskUID1    = "task00001"
+	taskUID2    = "task00002"
+	taskUID3    = "task00003"
+	taskUID4    = "task00004"
+	taskUnknown = "non_existing_taskID"
+
+	podName1     = "pod1"
+	podName2     = "pod2"
+	podName3     = "pod3"
+	podName4     = "pod4"
+	podNamespace = "yk"
+
+	nodeName1    = "node1"
+	nodeName2    = "node2"
 	fakeNodeName = "fake-node"
+
+	queueNameA       = "root.a"
+	queueNameB       = "root.b"
+	queueNameC       = "root.c"
+	queueNameUnknown = "root.other"
+
+	annoAppID1 = "yunikorn-test-00001"
+	annoAppID2 = "yunikorn-test-00002"
+	annoAppID3 = "yunikorn-test-00003"
+	annoAppID4 = "yunikorn-test-00004"
+	annoAppID5 = "yunikorn-test-00005"
```

Comment:
I haven't investigated the test deeply, but these look unnecessary. The appIDs should be unique, but the value really doesn't matter as long as they are unique. Just drop these and use `appID1`, `appID2`, etc.

Reply:
Many thanks for the suggestion. Update the code for annoAPPID1-5, run the `make test` , it passed.